inc/admin-pages/class-customer-edit-admin-page.php

```
@@ -444,10 +480,25 @@
 
 			$options = array_merge(['' => '--'], $options);
 
+			// Add simple delete link for orphaned fields (those without form reference)
+			$delete_link = '';
+			if ( ! $form) {
+				$delete_url = add_query_arg([
```

PHPStan (2.1.17)

Function add_query_arg not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(function.notFound)